content/post/2019-11-16-robot-framework-for-network-testing.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2018-11-16T00:00:00-00:00
+description: "Automated testing made easy and efficient with Robot framework"
```

Comment:
Please add draft tag via https://gohugo.io/getting-started/usage/

Reply:
Added the draft tag in the latest commit

Reply:
@Mierdin I've added draft: true to the top section of the blog. Is this what you meant?